Lib/test/test_getpass.py

```diff
@@ -14,6 +14,12 @@
 except ImportError:
     pwd = None
 
+try:
+    import msvcrt
+
+except ImportError:
+    mscvrt = None
```

Comment:
```suggestion
try:
    import msvcrt
except ImportError:
    mscvrt = None
```

Reply:
Also, you can stick the `try` block to the block above (see how `termios` and `pwd` have been imported)